### PR TITLE
unit tests for advanced_viewmodel and channel_add_people viewmodel

### PR DESCRIPTION
**Pull request by alecxchuk**
all_threads_viewmodel.dart is empty so i didnt write any test

https://user-images.githubusercontent.com/48722111/135700926-534b8043-139d-4d67-92f8-5499ef7d17dc.mp4

### CODE REVIEW

**Comment by alecxchuk** [comment]
@pauleke65

File: test/view_models/advanced_viewmodel_test.dart [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      });
+    });
+
+    group('toggleDisplayTypingIndicators -', () {
+      test(
+          'When toggleDisplayTypingIndicators is called should notifyListeners to rebuild UI',
+          () {
+        var model = _getModel();
+        bool called = false;
+        model.addListener(() {
```
Comment (pauleke65) [inline test/view_models/advanced_viewmodel_test.dart:59]: why is this here
if ```model.toggleDisplayTypinIndicators(true);``` is here

Comment (alecxchuk) [reply test/view_models/advanced_viewmodel_test.dart:59]: It's checking that the listener rebuilds the UI. I don't quite get your question. 
@pauleke65

Comment (pauleke65) [reply test/view_models/advanced_viewmodel_test.dart:59]: i meant your assigning true to called in the add listener
and also passing it through the display typing indicators.
so which is actually changing the value

**Review by pauleke65 — DISMISSED** [review]
LG

**Comment by alecxchuk** [comment]
@pauleke65 @YenumJay

**Review by jeremiahseun — APPROVED** [review]
LGTM

File: test/view_models/channel_add_people_viewmodel_test.dart [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        var searchWord = 'zuri';
+        model.onSearchUser(searchWord);
+        expect(
+            model.matchingUsers
+                .every((user) => user.userName!.contains(searchWord)),
+            true);
+      });
+    });
+
+    /*group('onFetchedUser -', () {
```
Comment (pauleke65) [inline test/view_models/channel_add_people_viewmodel_test.dart:77]: remove unused codes

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

File: test/view_models/channel_add_people_viewmodel_test.dart (hunk 2 above)
Comment (alecxchuk) [reply test/view_models/channel_add_people_viewmodel_test.dart]: Done that

**Review by pauleke65 — APPROVED** [review]
LGTM

**Review by jeremiahseun — APPROVED** [review]

**Comment by YenumJay** [comment]
All good
